### ISSUE TITLE
When and when not to show shared mapping

### ISSUE THREAD

**Issue by philbarker** [t3-innovation-network/desm#454]
When you are not logged in and click on "View Shared Mappings" from the home screen, the drop-down on resulting screen offers a choice of all mappings regardless of whether there are any mappings to see.

It should not show mappings when:
* the configuration profile is not Active
* there are no published mappings

When you are logged in as administrator and and click on "View Shared Mappings" from the home screen you get to see nothing. It makes no sense that someone who is logged in sees less that someone who isn't.

When you are logged in as a mapper, you get put straight through to the project that you have logged in for, which is a little confusing.

**Comment by philbarker** [comment]
On https://tool.desmsolutions.org/mappings-list do not show mappings when
- [ ] the configuration profile is not active
- [ ] no mappings are published

**Comment by philbarker** [comment]
I checked that the profiles listed in the drop down all had published mappings and that if I unpublished a mapping in one it was no longer listed. 

There seems to be something stopping me from unpublishing a mapping an then republishing it, it could be a bug, it could be that I corrupted a profile: I will check.

I think that the view mapping page still shows options for showing alignments of properties from standards for which none are published. That's minor compared to the main chose mapping option drop down so I shall move that to another page.  I will raise it as a separate issue.

Moving this to passed.
